lib/compiler/semantic.js

```diff
@@ -21,6 +23,8 @@ var SemanticPass = Base.extend({
 
     initialize: function(options) {
         this.now = options.now;
+        this.beginning = moment(-Infinity);
```

Comment:
This is consistent with the current behavior in which we process `beginning` by calling `new JuttleMoment(-Infinity)` which in turn calls `moment(-Infinity)`.

As we discussed separately, there's a potential argument to make that we shouldn't use infinite moments for these expressions, but my goal for this PR was not to make semantic changes to the language but simply to clean up the code and reference the separate library, so I don't want to make the behavior change yet.